src/service-override/extensionGallery.ts

```diff
@@ -0,0 +1,55 @@
+import { ExtensionRecommendationNotificationService } from 'vs/workbench/contrib/extensions/browser/extensionRecommendationNotificationService'
+import { ExtensionTipsService } from 'vs/platform/extensionManagement/common/extensionTipsService'
+import { ExtensionManagementService } from 'vs/workbench/services/extensionManagement/common/extensionManagementService'
+import 'vs/workbench/contrib/extensions/browser/extensions.contribution'
+import 'vs/workbench/contrib/extensions/browser/extensions.web.contribution'
+import 'vs/workbench/contrib/logs/common/logs.contribution'
+import { IRemoteUserDataProfilesService, RemoteUserDataProfilesService } from 'vs/workbench/services/userDataProfile/common/remoteUserDataProfiles'
+import { ExtensionEnablementService } from 'vs/workbench/services/extensionManagement/browser/extensionEnablementService'
+import 'vs/workbench/services/extensionManagement/browser/extensionBisect'
+
+export default function getServiceOverride (): IEditorOverrideServices {
+  return {
+    [ILocaleService.toString()]: new SyncDescriptor(WebLocaleService, [], false),
+    [IExtensionGalleryService.toString()]: new SyncDescriptor(ExtensionGalleryService, [], true),
+    [IGlobalExtensionEnablementService.toString()]: new SyncDescriptor(GlobalExtensionEnablementService, [], true),
+    [IExtensionsWorkbenchService.toString()]: new SyncDescriptor(ExtensionsWorkbenchService, [], true),
+    [IExtensionManagementServerService.toString()]: new SyncDescriptor(ExtensionManagementServerService, [], true),
+    [IExtensionRecommendationsService.toString()]: new SyncDescriptor(ExtensionRecommendationsService, [], true),
+    [IExtensionRecommendationNotificationService.toString()]: new SyncDescriptor(ExtensionRecommendationNotificationService, [], true),
+    [IWebExtensionsScannerService.toString()]: new SyncDescriptor(WebExtensionsScannerService, [], true),
+    [IExtensionIgnoredRecommendationsService.toString()]: new SyncDescriptor(ExtensionIgnoredRecommendationsService, [], true),
+    [IIgnoredExtensionsManagementService.toString()]: new SyncDescriptor(IgnoredExtensionsManagementService, [], true),
+    [IExtensionManifestPropertiesService.toString()]: new SyncDescriptor(ExtensionManifestPropertiesService, [], true),
+    [IExtensionManagementService.toString()]: new SyncDescriptor(ExtensionManagementService, [], true),
+    [IBuiltinExtensionsScannerService.toString()]: new SyncDescriptor(BuiltinExtensionsScannerService, [], true),
+    [IWorkspaceExtensionsConfigService.toString()]: new SyncDescriptor(WorkspaceExtensionsConfigService, [], true),
+    [IRemoteExtensionsScannerService.toString()]: new SyncDescriptor(RemoteExtensionsScannerService, [], true),
+    [IExtensionTipsService.toString()]: new SyncDescriptor(ExtensionTipsService, [], true),
+    [IRemoteUserDataProfilesService.toString()]: new SyncDescriptor(RemoteUserDataProfilesService, [], true),
```

Comment:
is it related to extensions?

Reply:
Yes, it's required to know which extension to load from the server.